### DISCUSSION TITLE
在暴力猴下是不是无法直接debug

### DISCUSSION THREAD

**Discussion by FoundTheWOUT** [the1812/Bilibili-Evolved#1802] [综合讨论 / General]
要起个本地服务? 这……
https://violentmonkey.github.io/api/metadata-block/#require
https://github.com/violentmonkey/violentmonkey/issues/430
![截屏2021-05-01 下午9 49 07](https://user-images.githubusercontent.com/47922588/116784496-0f7d0f00-aac7-11eb-9536-50794d612c91.png)

**Comment by the1812** [comment]
是的 调试还是 Tampermonkey 方便些
